projects.go

```diff
@@ -903,3 +903,37 @@ func (s *ProjectsService) UploadFile(pid interface{}, file string, options ...Op
 
 	return uf, resp, nil
 }
+
+// ListProjectForksOptions represents the available ListProjectForks() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/projects.html#list-project-hooks
+type ListProjectForksOptions struct {
+	ListProjectsOptions
+	WithIssuesEnabled        bool `url:"with_issues_enabled" json:"with_issues_enabled"`
+	WithMergeRequestsEnabled bool `url:"with_merge_requests_enabled" json:"with_merge_requests_enabled"`
+}
+
+// ListProjectForks gets a list of project forks.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/projects.html#list-project-hooks
```

Comment:
The doc URL should be `https://docs.gitlab.com/ce/api/projects.html#list-forks-of-a-project`